README.md

```diff
@@ -1,70 +1,33 @@
 # Catalogue Pipeline
 
-| CI Pipeline       | Status                                                                                                                                                                    |
-|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Build             | [![Build status](https://badge.buildkite.com/0ca819db1215b66ecb17019d8ee5331d8e537094d051141219.svg?branch=main)](https://buildkite.com/wellcomecollection/catalogue-pipeline)   |
+[![Build status](https://badge.buildkite.com/0ca819db1215b66ecb17019d8ee5331d8e537094d051141219.svg?branch=main)](https://buildkite.com/wellcomecollection/catalogue-pipeline)
 
-## Purpose
+## Overview
 
-Making Wellcome Collection's catalogue open, accessible and
-discoverable.
+If you are new to the catalogue pipeline the [GitBook documentation](https://docs.wellcomecollection.org/catalogue-pipeline) is a good place to start.
 
-The catalogue consists of multiple sources including:
-* Library holdings
-* Archives and manuscripts
-* Born digital content
-* Images from what was previously wellcomeimages.org
+## Developing
 
-As and when these sources are made available digitally, we will consume
-them via [our pipeline](./pipeline), unify them into a
-[single model](./common/internal_model) and make them discoverable via
-our [API](https://github.com/wellcomecollection/catalogue-api).
+Information for developers working on the catalogue-pipeline.
 
-**Interested in how we make these services?**
+### Deploying
 
-[Take a look at our documentation on the design and decision making
-processes of the services within the catalogue repo][catalogue docs].
+We deploy catalogue-pipeline services using the [weco-deploy](https://github.com/wellcomecollection/weco-deploy) tool.
 
-**Interested in making use of our data to build your own products or
-use in your research?**
+### Things you might want to do
 
-[Take look at our developer documentation][api developer docs] or
-[go straight to our API][api].
+#### Reindexing
 
-**Interested in other parts of the Wellome Collection digital platform
-works?**
+If the [internal_model](./common/internal_model) has been changed you will want to update the information stored by the pipeline to match that model.
 
-[Take a look at our Platform repo][platform repo]
+A reindex operation runs the source data from the [adapters](docs/adapters/README.md) through the pipeline causing it to be re-transformed / matched & merged as appropriate.
```

Comment:
Transformed makes sense, matched merged could use a link to the docs on that... which I don't htink exist.

Reply:
I agree there should be some explanation of matching and merging in these docs, but I won't add that in this PR.